packages/hlsjs-playback/README.md

```
@@ -42,50 +42,51 @@ or as an npm package:
+  },
+  playback: {
+    extrapolatedWindowNumSegments: 2,
+    triggerFatalErrorOnResourceDenied: false,
+    hlsjsConfig: {
+      // hls.js specific options
+    }
+  }
+})
 ```
 
 #### hlsUseNextLevel
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)